tools/api-markdown-documenter/src/api-item-transforms/test/Transformation.test.ts

```diff
@@ -18,26 +18,18 @@ import {
 } from "@microsoft/api-extractor-model";
 import { expect } from "chai";
 
-import type { ApiDocument } from "../../ApiDocument.js";
-import type { Section } from "../../mdast/index.js";
-import { createSection } from "../../mdast/index.js";
-import { getHeadingForApiItem } from "../ApiItemTransformUtilities.js";
-import { apiItemToSections } from "../TransformApiItem.js";
+import type { MarkdownDocument } from "../../ApiDocument.js";
```

Comment:
Note for reviewers: I'm sorry... These are not great tests and should probably be replaced with simpler unit tests in the future.

The changes here result from the transformation layer now outputting "normalized" markdown trees, instead of trees containing our special `Section` and `SectionHeading` nodes, which creates substantial diffs, and refactoring them to use the same "as document" transformation path (which added breadcrumbs and root headings).